.gitignore

```diff
@@ -1,3 +1,5 @@
+.gitignore
```

Comment:
ignore the .gitignore? why?

Reply:
have some files in my local repository that's in my .gitignore that i don't want to be public, not a big deal really and i can revert it back to normal. don't know why git still uploaded the .gitignore file though